app/src/organisms/Devices/PipetteCard/FlexPipetteCard.tsx

```diff
@@ -0,0 +1,282 @@
+          }
+          instrumentDiagramProps={{
+            pipetteSpecs: pipetteModelSpecs,
+            mount,
+          }}
+          banner={
+            attachedPipette?.ok &&
+            attachedPipette.data.calibratedOffset?.last_modified == null ? (
+              <Banner type="error" marginBottom={SPACING.spacing4} width="100%">
+                {isEstopNotDisengaged ? (
+                  <StyledText as="p">
+                    {t('calibration_needed_without_link')}
+                  </StyledText>
+                ) : (
+                  <Trans
+                    t={t}
+                    i18nKey={'calibration_needed'}
+                    components={{
+                      calLink: (
+                        <StyledText
+                          as="p"
+                          css={BANNER_LINK_CSS}
+                          onClick={handleCalibrate}
+                        />
+                      ),
+                    }}
+                  />
+                )}
+              </Banner>
+            ) : null
+          }
+          label={
+            attachedPipetteIs96Channel
+              ? t('both_mounts')
+              : t('mount', {
+                  side: mount === LEFT ? t('left') : t('right'),
+                })
+          }
+          menuOverlayItems={menuOverlayItems}
+          isEstopNotDisengaged={isEstopNotDisengaged}
+        />
+      ) : null}
+      {attachedPipette?.ok === false ||
+      (subsystemUpdateData != null && pollForSubsystemUpdate) ? (
+        <InstrumentCard
+          label={i18n.format(t('mount', { side: mount }), 'capitalize')}
+          css={INSTRUMENT_CARD_STYLE}
+          description={t('instrument_attached')}
+          banner={
+            <Banner
+              type={subsystemUpdateData != null ? 'warning' : 'error'}
+              marginBottom={SPACING.spacing4}
+            >
+              <Trans
+                t={t}
+                i18nKey={
+                  subsystemUpdateData != null
+                    ? 'firmware_update_occurring'
+                    : 'firmware_update_needed'
+                }
+              />
+            </Banner>
+          }
+          isEstopNotDisengaged={isEstopNotDisengaged}
+        />
+      ) : null}
+      {showDropTipWizard && pipetteModelSpecs != null ? (
+        <DropTipWizard
+          robotType={FLEX_ROBOT_TYPE}
+          mount={mount}
+          instrumentModelSpecs={pipetteModelSpecs}
+          closeFlow={() => setShowDropTipWizard(false)}
+        />
+      ) : null}
+      {attachedPipette?.ok && showAboutPipetteSlideout && (
+        <AboutPipetteSlideout
+          pipetteId={attachedPipette.serialNumber}
+          pipetteName={pipetteDisplayName ?? attachedPipette.instrumentName}
+          firmwareVersion={attachedPipette.firmwareVersion}
+          isExpanded={showAboutPipetteSlideout}
+          onCloseClick={() => setShowAboutPipetteSlideout(false)}
+        />
+      )}
```

Comment:
nit, to keep the same pattern for easier readability:

```suggestion
      {attachedPipette?.ok && showAboutPipetteSlideout ? (
        <AboutPipetteSlideout
          pipetteId={attachedPipette.serialNumber}
          pipetteName={pipetteDisplayName ?? attachedPipette.instrumentName}
          firmwareVersion={attachedPipette.firmwareVersion}
          isExpanded={showAboutPipetteSlideout}
          onCloseClick={() => setShowAboutPipetteSlideout(false)}
        />
      ) : null}
```